src/usr/local/www/services_unbound_host_edit.php

```diff
@@ -281,6 +287,8 @@ function hosts_sort() {
 	'fa-plus'
 ))->removeClass('btn-primary')->addClass('btn-success addbtn');
 
+$section->setHelp("If the host can be accessed using multiple names, then enter any other names for the host which should also be overridden.");
```

Comment:
This doesn't work.
`Fatal error: Call to undefined method Form_Section::setHelp() in /usr/local/www/services_unbound_domainoverride_edit.php on line 149 Call Stack: 0.0015 231544 1. {main}() /usr/local/www/services_unbound_domainoverride_edit.php:0 PHP ERROR: Type: 1, File: /usr/local/www/services_unbound_domainoverride_edit.php, Line: 149, Message: Call to undefined method Form_Section::setHelp()`